Makefile

```diff
@@ -26,11 +26,15 @@ imagebuilder:
 golangci-lint:
 	@type -p golangci-lint > /dev/null || go get github.com/golangci/golangci-lint/cmd/golangci-lint
 
+build-impl:
+	go build $(BUILD_OPTS) -o bin/cluster-logging-operator ./cmd/manager
 
-build:
-	go build -o bin/cluster-logging-operator ./cmd/manager
+build: build-impl
 
-run:
+build-debug: BUILD_OPTS=-gcflags=all="-N -l"
+build-debug: build-impl
```

Comment:
This could be simplified to keep the number of make targets exposed to the user small.
```
build:
 go build $(BUILD_OPTS)

build-debug:
  $(make) build BUILD_OPTS=...
```

WDYT?

Reply:
I'm usually allergic to recursive make, but in this case it doesn't really matter. Let's go with fewer make targets.